packages/main/src/Input.ts

```diff
@@ -1034,11 +1034,12 @@ class Input extends UI5Element implements SuggestionComponent, IFormInputElement
 		this.fireEvent("select", {});
 	}
 
-	_handleInput(e: InputEvent | CustomEvent<InputEventDetail>) {
+	_handleInput(e: InputEvent | CustomEvent<InputEventDetail> | Event) {
 		const inputDomRef = this.getInputDOMRefSync();
 		const emptyValueFiredOnNumberInput = this.value && this.isTypeNumber && !inputDomRef!.value;
 		const eventType: string = (e as InputEvent).inputType
-			|| (e.detail as InputEventDetail).inputType
+			|| (e as Event).type
```

Comment:
I don't think we need to check for e.type as it does not make any difference to the code at all. eventType is used to check whether it is included in allowedEventTypes, which in this case will be always false. I think it's better to just check if e.detail exists.